### DISCUSSION TITLE
Help usage...

### DISCUSSION THREAD

**Discussion by z3r01z3r01** [DirectoryLister/DirectoryLister#618] [Help & Support] [answered]
[SOLVED]
Hello. Try using this 'php -S localhost:8080` to start the server but I do not see any file on the listing... How to configure a folder to list files?
what I'm doing wrong .. try to activate the debuggin mode but I could not...
I want to use it to share files on my local network
![Screenshot_2021-04-30_22_57_27](https://user-images.githubusercontent.com/83483229/116767648-dd58a800-aa07-11eb-970c-64d1882b33d5.png)

**Comment by PHLAK** [comment, marked as answer]
Try enabling [debug](https://docs.directorylister.com/configuration/app-config-reference#debug) and searching for errors on Google. If you still are having issues post them here as well.

**Reply by z3r01z3r01** [reply]
had to install DOM
I followed the instructions on your page, thank you.. good job :)
